### ISSUE TITLE
Add Adoption information to Accounts

### ISSUE THREAD

**Issue by allisonhowell** [openstax/bit#282]
# Description
There are a few benefits to having adoption information in Accounts. 
- As we begin to track more user behavior in accounts, it's important to know their adoption status.
- We can set up a custom dimension in Google Analytics that would give us greater insight into what adopters vs. non-adopters are doing on the website. See this card: https://app.zenhub.com/workspaces/openstax-bit-5bacfe3f4b5806bc2bea3764/issues/openstax/bit/255 
- We could also continue to refine and improve the hero journey page that was created for the marketing team, which they hope to become a page that users can regularly visit to collect their badges, etc. 

Accounts could make a daily call to Salesforce to get the adoption information and store and serve it with the rest of the information for the account.
# Acceptance Criteria
- A users' adoption status in in Accounts. 
# Acceptance Tests

derive these from "Acceptance Criteria" during "Design" phase

**title**: template  
> GIVEN something  
> AND something else  
> WHEN something  
> AND something else  
> THEN something  
> AND something else  

# Checklist for Done
1A-DEFINE
- [ ] Description is complete 
- [ ] High-level natural language acceptance tests are complete
- [ ] If UX design is needed then the UX label is added
- [ ] If UX design is needed then the UX LoFi Design is completed, approved by the Product Manager, and added to the issue.
- [ ] A high-level solution (development approach for the code change) is added to the issue
- [ ] If prototype code is generated, it is linked to the issue.
- [ ] If tech briefs are generated, they are linked to the issue.

1B-DECOMP
- [ ] The work is broken into reasonable and consistent work items.
- [ ] Story issues are created for the work items, with Description, Acceptance, the label 'story', and an Estimate.
- [ ] The original issue is changed to an epic and the child story issues are linked to it.  The estimate is removed.
- [ ] The story epic and child work item issues are moved into 1B-DECOMP DONE.
- [ ] If an issue is included in a product phase, then Release is selected.

1C-DESIGN
  - [ ] The UX HiFi Design is complete, approved by the Product Manager, and added to the issue.
  - [ ] Distilled acceptance tests are added to the issue.
  - [ ] Automated testing criteria are added to issue.
  - [ ] The solution is updated.

2-CODE
  - [ ] A pull request is opened, the automated pull request checks pass, and the change has been approved by other developers.
  - [ ] The acceptance tests are finalized in "Given / When / Then" format.
  - [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.

4A-UX REVIEW
  - [ ] The pull request is merged into master, and the change branch is deleted.

5A-FUNCT VER
  - [ ] All the acceptance tests for this issue pass.

5C-REGRESSION
  - [ ] The test plan in testrail passes.
  - [ ] Blessing from Product Manager that the release is good to go.

5D-READY TO BE DEPLOYED
  - [ ] Production is updated and verified.
  - [ ] All issues in the release are closed.

**QA needs checklist:**
  - [ ] Acceptance Criteria are complete
  - [ ] Milestone should be tagged for PR's 
  - [ ] Milestone should be tagged for issues
  - [ ] Content update needed
  - [ ] Content update complete

**Deployment needs**
  - [ ] Pre-deployment tasks
  - [ ] Data migrations 
  - [ ] Post-deployment tasks

**Comment by allisonhowell** [comment]
Waiting on user id tracking

**Comment by allisonhowell** [comment]
UserID tracking doesn't provide any insight, so we need to move forward with this.
